pkg/storage/storagewrappers/combinedtuplereader_test.go

```diff
@@ -0,0 +1,896 @@
+package storagewrappers
+
+import (
+	"context"
+	"errors"
+	"reflect"
+	"testing"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+
+	"github.com/openfga/openfga/internal/mocks"
+	"github.com/openfga/openfga/pkg/storage"
+)
+
+var (
+	testTuples = map[string]*openfgav1.Tuple{
+		"user:11#member#group:1": {
+			Key: &openfgav1.TupleKey{
+				User:     "user:11",
+				Relation: "member",
+				Object:   "group:1",
+			},
```

Comment:
Tip: you don't have to manually type out the TupleKey, you can use this helper: https://github.com/openfga/openfga/blob/f3be13c8969feffa443eed902b3dc377a8b9b8f4/pkg/tuple/tuple.go#L362-L364 :)

Reply:
good to know! however, I can't inline the call without error handling

Reply:
fwiw we have the Must* equivalent https://github.com/openfga/openfga/blob/f6d055231a97ffa50e4755f4755ef935dc9e1ab6/pkg/tuple/tuple.go#L340